spec/controllers/bulk_ingest_controller_spec.rb

```diff
@@ -82,6 +109,69 @@
       end
     end
 
+    context "with files hosted on a cloud-storage provider" do
+      let(:selected_files) do
+        {
+          "0" => { "url" => "https://www.cloud.com/files/1.tif?alt=media", "file_name" => "1.tif", "file_size" => "100", "auth_header" => { "Authorization" => "Bearer secret" } },
+          "1" => { "url" => "https://www.cloud.com/files/2.tif?alt=media", "file_name" => "2.tif", "file_size" => "100", "auth_header" => { "Authorization" => "Bearer secret" } }
```

Comment:
I know these URLs won't be retrieved because the BrowseEverythingJob is stubbed, but can you change them to use `example.com`?